verification/src/compiler/compilers.rs

```diff
@@ -50,3 +67,131 @@ impl<T: VersionList> VersionList for Compilers<T> {
         self.fetcher.all_versions()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::solidity::{CompilerFetcher, Releases};
+    use std::{env::temp_dir, str::FromStr};
+
+    use async_once_cell::OnceCell;
+    use ethers_solc::artifacts::{Source, Sources};
+    use std::default::Default;
+
+    async fn global_compilers() -> &'static Compilers<CompilerFetcher> {
+        static COMPILERS: OnceCell<Compilers<CompilerFetcher>> = OnceCell::new();
+        COMPILERS
+            .get_or_init(async {
+                let url = "https://solc-bin.ethereum.org/linux-amd64/list.json"
```

Comment:
*somewhere in the world a macos user is crying*

why not to read from config?

Reply:
Hmm, makes sense

> why not read from config?

Because config is broader entity which is not related to Compilers module at all as required data from config is passed to Compilers separately as parameters. 

I believe it may make sense to extract the `DEFAULT_COMPILER_LIST` constant in a separate module and import it in Config as well as in Compilers

Reply:
right, but why in seprate module? maybe here, in `compilers.rs` file?

Reply:
Because we reuse it in both `compilers.rs` and `config.rs`

Reply:
okay, its deal :)